### DISCUSSION TITLE
可以利用github actions的cache缓存依赖库吗

### DISCUSSION THREAD

**Discussion by RimuruChan** [xmake-io/xmake#3516] [Q&A]
每次拉去安装耗时有点久

**Comment by waruqi** [comment]
参考 https://github.com/NazaraEngine/NazaraEngine/blob/3c2172103ae78c1be5ef7d23b9ff4f832e2201c7/.github/workflows/linux-build.yml#L66

**Reply by RimuruChan** [reply]
我是windows，把~/AppData/Local/.xmake 和 ./xmake 缓存了可以吗

**Reply by waruqi** [reply]
上面那个 也有 windows ci，自己参考下
